spacewalk/package/spacewalk.spec

```
@@ -110,20 +113,15 @@ Requires:       spacewalk-backend-sql-postgresql
 Requires:       spacewalk-java-postgresql
 Requires:       perl(DBD::Pg)
 %if 0%{?suse_version}
-# Actual version set by prjconf, default is 14
-%{!?postgresql_version_min: %global postgresql_version_min 14}
-%{!?postgresql_version_max: %global postgresql_version_max 15}
-Requires:       postgresql-contrib-implementation >= %{postgresql_version_min}
 Requires:       postgresql-implementation >= %{postgresql_version_min}
-Conflicts:      postgresql-contrib-implementation > %{postgresql_version_max}
-Conflicts:      postgresql-implementation > %{postgresql_version_max}
+Requires:       postgresql-implementation <= %{postgresql_version_max}.9999
+Requires:       postgresql-contrib-implementation >= %{postgresql_version_min}
+Requires:       postgresql-contrib-implementation <= %{postgresql_version_max}:.9999
```

```suggestion
Requires:       postgresql-contrib-implementation <= %{postgresql_version_max}.9999
```

I think this `:` is extra here, right?

---

Thanks for spotting... the accidental `i` and `:` of vi ...